src/routes/promises/+page.server.ts

```diff
@@ -36,26 +42,36 @@ export interface PromisesByCategory {
 	count: number;
 }
 
+export type AssemblyMember = ReturnType<typeof getAssemblyMembers>[number];
+
 const CURRENT_CABINET_ASSEMBLY_ID = 'คณะรัฐมนตรี-64';
+const PREVIOUS_CABINET_ASSEMBLY_ID = 'คณะรัฐมนตรี-63';
```

Comment:
I think we should get rid of this hard code. Can you remove this constant here and then in the `load()`, you filter only "คณะรัฐมนตรี" name from the `assemblies`, sorted by term and then assign the latest 2 as current and previous cabinet?